core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -320,6 +320,7 @@ class GroupCoordinator(val brokerId: Int,
             } else {
               val member = group.get(memberId)
               removeHeartbeatForLeavingMember(group, member)
+              info(s"Member ${member.memberId} in group ${group.groupId} has left, removing it from the group")
```

Comment:
Since this is a common case when a consumer is shutdown, maybe debug level is sufficient?